packages/cli/tsconfig.json

```diff
@@ -1,27 +1,14 @@
 {
+	"extends": "../../tsconfig.json",
 	"compilerOptions": {
-		"lib": [
-			"es2017",
-			"ES2020.Promise"
-		],
+		"outDir": "dist",
 		"types": [
 			"node",
 			"jest"
 		],
-		"module": "commonjs",
-		"noImplicitAny": true,
-		"removeComments": true,
-		"strictNullChecks": true,
-		"forceConsistentCasingInFileNames": true,
-		"noImplicitReturns": true,
-		"preserveConstEnums": true,
-		"declaration": true,
-		"outDir": "./dist/",
-		"target": "es2017",
-		"sourceMap": true,
-		"emitDecoratorMetadata": true,
-		"experimentalDecorators": true,
-		"esModuleInterop": true
+		// TODO: remove all options below this line
```

Comment:
Leaving these `TODO` here because I'll be sending separate PRs removing these exceptions.